interp/type.go

```diff
@@ -636,6 +636,47 @@ func (t *itype) finalize() (*itype, error) {
 	return t, err
 }
 
+// identical returns true if the given type is identical to the receiver one
+func (t *itype) identical(o *itype) bool {
```

Comment:
maybe you can use `equal` or `isEqual` instead of `identical`.

Reply:
renamed `identical` to `equal`